public/tutorials/cross-dom-bridge-erc20.js

```
@@ -1,93 +1,185 @@
+    functionName: 'balanceOf',
+    args: [account.address]
+  });
+  console.log(`L1 Balance: ${formatEther(l1BalanceAgain)}`);
+
+  const l2BalanceAfterDeposit = await await publicClientL2.readContract({
+    address: l2Token,
+    abi: erc20ABI,
+    functionName: 'balanceOf',
+    args: [account.address]
+  });
+  console.log(`L1 Balance after deposit: ${formatEther(l2BalanceAfterDeposit)}`);
```

ESLint

[error] Extra semicolon.

(semi)